tests/test_basic.py

```
@@ -0,0 +1,31 @@
+"""
+Basic tests for the MCP Agent API.
+"""
+import sys
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)